### ISSUE TITLE
Bring back support/android/apk/project.properties

### ISSUE THREAD

**Issue by aneeshusa** [servo/servo#11876]
See discussion at https://github.com/servo/servo/pull/11687#discussion-diff-66534511L37.

This file was removed in #11278, but the automated tooling will rewrite the other Android files to refer to it, so it should be brought back, ideally without duplicating the `android-18` value and instead reading it from the environment.

cc @larsbergstrom

**Comment by larsbergstrom** [comment]
@rzambre If this file got created in your successful build, fixing this would really help :-)

**Comment by lumiscosity** [comment]
I don't think this is relevant anymore? I played around with Android builds today and I never saw anything about a project.properties... The proguard pointers and target version are in the .kts files, so this probably got dropped in the migration, if it was ever left behind in the first place.
